packages/tools/visual-tests/playwright.config.js

```diff
@@ -63,4 +63,5 @@ export default defineConfig({
 		url: URL,
 		reuseExistingServer: false,
 	},
+	snapshotPathTemplate: `{snapshotDir}/theme-${(process.env.THEME_EXPORT || 'default').toLocaleLowerCase()}/{arg}-{projectName}-{platform}{ext}`,
```

Comment:
Findet die Funktion `snapshotPath` in theme-snapshots.spec.js trotzdem noch Anwendung oder kann sie entfernt werden?